generator/lib/src/code_chunks.dart

```diff
@@ -34,7 +34,8 @@ class CodeChunks {
     return """
       EntityDefinition<${name}>(
         model: model.getEntityByUid(${entity.id.uid}),
-        toOneRelations: ($name inst) => [${toOneRelationsList(entity).join(',')}],
+        toOneRelations: ($name inst) => ${toOneRelations(entity)},
+        toManyRelations: ($name object) => ${toManyRelations(entity)},
```

Comment:
Other places in the definition call the entity instance `inst`, so maybe rename `object` to `inst`?

Reply:
I'm aware and plan to rename all of the calls to object (that's how it's used in other parts of the code, like `Box` ...), just didn't want to do it before the previous PRs got merged.